lib/config/presets/codecommit/index.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { TextEncoder } from 'web-encoding';
+import { Platform, setPlatformApi } from '../../../modules/platform';
+import { PRESET_DEP_NOT_FOUND } from '../util';
+import * as codeCommit from '.';
+
+jest.unmock('../../../modules/platform');
```

Comment:
why this? shouldn't required for these tests.

Reply:
i changed this already yesterday and commited, why is it still here????

Reply:
forgot to push? 🙃

Reply:
see those commits yesterday 
9dc73a908c812c3e9acab1c2293edcedf3af0dd8
4dc0a9df8e0597d0aeb52d35c534fecd2ab37222
they are not showing in here for some reason

Reply:
should be outdated :/
see https://github.com/PhilipAbed/renovate/blob/codecommit/lib/modules/platform/codecommit/index.spec.ts

Reply:
sorry, my mistake, 
i tried to remove it, but now i receive an error when i try to set platform
![image](https://user-images.githubusercontent.com/42116482/191224138-a77c50a4-1114-43d0-8b5f-393a0eab166c.png)

Reply:
@viceice can we leave it as-is, unless you know a quick fix?

Reply:
this whole file should use the mocked platform and don't call through.

i don't see any benefits to test internal codecommit platform code here.

Reply:
@viceice the code doesn't pass tests because the functions are using the platform, look at other examples 

the other option is to NOT do tests at all and `istanbul ignore`

Reply:
Do we mock the platform for other platform presets?

Reply:
we don't need this tests at all, coverage should be ok without

Reply:
ok i'll add `istanbul ignore next` then and delete the whole test file